hal/include/hal/serial_api.h

```diff
@@ -296,6 +296,13 @@ int  serial_readable(serial_t *obj);
  */
 int  serial_writable(serial_t *obj);
 
+/** Check if the serial peripheral tx buffer is empty
```

Comment:
As you noted, empty in this case means (fifo empty and also shift register) ? Is it clear to a user who ports a new board?

Reply:
i will change the comment to make it more explicit

Reply:
i've updated function description here bdb44e2b54d21e1b8e552b29a4f3d37098fa4ab8